revision_data.h.in.cmake

```diff
@@ -9,7 +9,7 @@
  #define _SOURCE_DIRECTORY          R"(@CMAKE_SOURCE_DIR@)"
  #define _BUILD_DIRECTORY           R"(@BUILDDIR@)"
  #define _MYSQL_EXECUTABLE          R"(@MYSQL_EXECUTABLE@)"
- #define _FULL_DATABASE             "TDB_full_world_335.20031_2020_03_16.sql"
+ #define _FULL_DATABASE             "TDB_full_world_335.@rev_version_major@@rev_version_minor@@rev_version_patch@_@rev_database_date@.sql"
```

Comment:
does this mean the version naming will be changed ? Like what would 20031 become with the new system ?

Reply:
since https://github.com/TrinityCore/TrinityCore/commit/3358cc2a4772aedf6415b5ba7ca3fdbbf808da24#commitcomment-32815761 (scroll down at the comment at the bottom, it's a huge commit) we changed the TDB versioning to follow this syntax:
| Header | TDB | {branch} | . | {last 2 digits of the year} | {2 digits of the month} | {counter of TDB releases for the month  starting from 1} |
| --- | --- | --- | --- | --- | --- | --- |
| Today's release | TDB | 335 | . | 19 | 03 | 1 |
| Next week's release (if any) | TDB | 335 | . | 19 | 03 | 2 |
| Next month's release | TDB | 335 | . | 19 | 04 | 1 |

I would like to keep the {year}{month}{counter} format if possible without having to handle a major.minor.patch that for TC development really doesn't have much of a meaning (we don't have a stable release schedule and we dropped counting TDB with incremental numbers).

Is major.minor.patch a requirement by CPack ?